nabla/test/nablaCurve.ts

```diff
@@ -13,61 +13,61 @@ export default async function (environment: TestSuiteEnvironment) {
   const curve = await newNablaCurve(0, e(0.01, 18));
```

Comment:
The changes in this test suite resemble the changes in the [Nabla curve contract PR](https://github.com/0xamberhq/curve/pull/4) for the file `test/NablaCurve.t.sol`.